src/main/zip/plugin.xml

```diff
@@ -142,10 +141,16 @@
                      label="Login Key Secret"
                      description="The API key secret to authenticate with ASoC." />
       </property>
+      <property name="scanName" required="false">
+        <property-ui type="textBox"
+                     label="Scan Name"
+                     description="Enter a name for the scan. If empty, the Starting URL will be used." 
+                     default-value="${p:component.name} - ${p?:version.name}"/>
```

Comment:
Maybe wanna do ${p?:component.name} in the case that someone would kick this off from a generic process. So they don't get a failure to access a property. Also, maybe just do a space between component and version so that in the case neither property exists they don't just get a ' - ', and then trim the value later when you check if they've specified it.